### ISSUE TITLE
cartocss `Map` `background-color` does not support view::zoom

### ISSUE THREAD

**Issue by farfromrefug** [CartoDB/mobile-sdk#501]
if i set this:
```
Map {
background-color: linear([view::zoom], (5, #e6e4e0), (22, red))
}
```
Then i end up with a transparent background

**Comment by mtehver** [comment]
Indeed, this is not currently supported. I have to check if this can be easily implemented. Just being curious, what is your use case for this?

**Comment by farfromrefug** [comment]
@mtehver i am reproducing OSM style in cartocss. And they change the background color with view zoom.

**Comment by mtehver** [comment]
@farfromrefug This is now implemented in 'develop' branch (you need 'develop' branch from mobile-carto-libs also).

**Comment by farfromrefug** [comment]
@mtehver thanks will test it!

**Comment by mtehver** [comment]
Closing this, implemented in 4.4.7 release.
